algorithms/CPlusPlus/Arrays/fractional-knapsack.cpp

```diff
@@ -0,0 +1,77 @@
+// Description : Given weights and values of n items, put these items in a knapsack of capacity W to get the maximum total value in the knapsack.
+// We can break items for maximizing the total value of knapsack/
+
+// Algorithm Type: Divide & Conquer
+// Time Complexity: O(n*log(n))
+#include <bits/stdc++.h>
+using namespace std;
+class Item
+{
+public:

```

Comment:
If there was no private member in the class then why not go for struct at the first place ?